### PR TITLE
Wait before capture

### PR DESCRIPTION
**Pull request by chaim-aharonson-applitools**

### CODE REVIEW

File: coverage-tests.js [hunk 1]
```diff
@@ -2066,4 +2066,38 @@ test('should return browserInfo in getAllTestResults', {
   },
 })
 
-// #endregion
+test('should waitBeforeCapture eyesOpen', {
```
Comment (amitzur) [inline coverage-tests.js]: rename: `should waitBeforeCapture in open`

File: coverage-tests.js [hunk 2]
```diff
@@ -2066,4 +2066,38 @@ test('should return browserInfo in getAllTestResults', {
...
+    ]
+  },
+  test({ driver, eyes }) {
+    driver.visit('https://applitools.github.io/demo/TestPages/waitBeforeCapture')
+    eyes.open({ appName: 'Applitools Eyes SDK', viewportSize: { width: 600, height: 600 } })
+    eyes.check({isFully: true})
+    eyes.close()
+  },
+})
+test('should waitBeforeCapture eyesCheck', {
```
Comment (amitzur) [inline coverage-tests.js]: rename: `rename: `should waitBeforeCapture in check`

**Review by amitzur — APPROVED** [review]
I proposed small renaming because `eyesOpen` and `eyesCheck` exist in Cypress but not in other SDK's

File: java/overrides.js [hunk 3]
```diff
@@ -42,5 +42,6 @@ module.exports = {
     // REQUEST: {"name":"EyesManager.closeManager","key":"36cd5684-44c2-4b96-a217-803d1efc7981","payload":{"manager":{"applitools-ref-id":"4e73260e-70b8-49aa-a5d2-369246ded3b3"},"throwErr":false}}
     // RESPONSE: {"name":"EyesManager.closeManager","key":"36cd5684-44c2-4b96-a217-803d1efc7981","payload":{"result":{"results":[],"passed":0,"unresolved":0,"failed":0,"exceptions":0,"mismatches":0,"missing":0,"matches":0}}}
     'check window on mobile web android': {skip: true},
-
+    'should waitBeforeCapture eyesOpen': { skipEmit: true },
+    'should waitBeforeCapture eyesCheck': { skipEmit: true },
```
Comment (Yarden-Ingber) [inline java/overrides.js:46]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9080554)
```suggestion
    "should waitBeforeCapture eyesCheck": { skipEmit: true },
```

File: java/overrides.js [hunk 4]
```diff
@@ -42,5 +42,6 @@
     // REQUEST: {"name":"EyesManager.closeManager","key":"36cd5684-44c2-4b96-a217-803d1efc7981","payload":{"manager":{"applitools-ref-id":"4e73260e-70b8-49aa-a5d2-369246ded3b3"},"throwErr":false}}
     // RESPONSE: {"name":"EyesManager.closeManager","key":"36cd5684-44c2-4b96-a217-803d1efc7981","payload":{"result":{"results":[],"passed":0,"unresolved":0,"failed":0,"exceptions":0,"mismatches":0,"missing":0,"matches":0}}}
     'check window on mobile web android': {skip: true},
-
+    'should waitBeforeCapture eyesOpen': { skipEmit: true },
```
Comment (Yarden-Ingber) [inline java/overrides.js:45]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9080554)
```suggestion
    "should waitBeforeCapture eyesOpen": { skipEmit: true },
```

File: coverage-tests.js (hunk 1 above)
Comment (chaim-aharonson-applitools) [reply coverage-tests.js]: done

File: coverage-tests.js (hunk 2 above)
Comment (chaim-aharonson-applitools) [reply coverage-tests.js]: done

File: java/overrides.js [hunk 5]
```diff
@@ -42,5 +42,6 @@
     // REQUEST: {"name":"EyesManager.closeManager","key":"36cd5684-44c2-4b96-a217-803d1efc7981","payload":{"manager":{"applitools-ref-id":"4e73260e-70b8-49aa-a5d2-369246ded3b3"},"throwErr":false}}
     // RESPONSE: {"name":"EyesManager.closeManager","key":"36cd5684-44c2-4b96-a217-803d1efc7981","payload":{"result":{"results":[],"passed":0,"unresolved":0,"failed":0,"exceptions":0,"mismatches":0,"missing":0,"matches":0}}}
     'check window on mobile web android': {skip: true},
-
+    'should waitBeforeCapture in open': { skipEmit: true },
```
Comment (Yarden-Ingber) [inline java/overrides.js:45]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9080554)
```suggestion
    "should waitBeforeCapture in open": { skipEmit: true },
```

File: java/overrides.js [hunk 6]
```diff
@@ -42,5 +42,6 @@
...
+    'should waitBeforeCapture in check': { skipEmit: true },
```
Comment (Yarden-Ingber) [inline java/overrides.js:46]: Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9080554)
```suggestion
    "should waitBeforeCapture in check": { skipEmit: true },
```

**Pull request merged**